demisto_sdk/commands/common/hook_validations/pack_unique_files.py

```diff
@@ -79,8 +79,7 @@ class PackUniqueFilesValidator(BaseValidator):
 
     def __init__(self, pack, pack_path=None, validate_dependencies=False, ignored_errors=None, print_as_warnings=False,
                  should_version_raise=False, id_set_path=None, suppress_print=False, private_repo=False,
-                 skip_id_set_creation=False, prev_ver=main_branch, json_file_path=None, support=None,
-                 empty_readme_check=True):
+                 skip_id_set_creation=False, prev_ver=main_branch, json_file_path=None, support=None):
```

Comment:
intended change or an autoformatted indent?

Reply:
intended change. No one uses this flag. Vulture said so